.eslintrc.json

```diff
@@ -29,6 +32,7 @@
       "no-case-declarations": "warn",
       "no-useless-escape" : "warn",
       "no-prototype-builtins": "warn",
-      "no-unused-vars": ["warn", { "argsIgnorePattern": "^_" }]
+      "no-unused-vars": ["warn", { "argsIgnorePattern": "^_" }],
+      "no-undef": "off"
```

Comment:
NOTE 'no-undef' rule is off because the most vscode-webview's client-side codes 'document' and 'window'